### DISCUSSION TITLE
PR and PG matrices empty with GluC digest

### DISCUSSION THREAD

**Discussion by ambgrav** [vdemichev/DiaNN#954] [Q&A]
Hello,

I recently did an experiment where I only used a GluC protease, and after processing the raw files in FragPipe I found that the DIA-NN pr and pg matrices were empty while the main report still had peptides. I am interested in the peptide intensity of each replicate, which is why I was planning on using the pr matrix. Do you have any input on how I can better process this data or why these files may be empty?

Thank you very much,
Amber

**Comment by vdemichev** [comment]
Hi Amber,

They are empty if no protein passes the q-value filters.
In general, we recommend always using the main report, this gives you full transparent control of any filtering being done. It also allows you to easily spot situations, for example, when protein annotation could simply be missing from the spectral library.

Best,
Vadim
